.gitignore

```diff
@@ -9,3 +9,8436 @@ pyvenv.cfg
 
 venv
 __pycache__
```

Comment:
Revert changes on this file, if you want to ignore the inference_env folder you can add inference_env folder, but it's better if you name the folder to `venv` so it's ignored as per the instruction on line 10